tracing-subscriber/src/fmt/fmt_layer.rs

```diff
@@ -290,6 +290,31 @@ where
         }
     }
 
+    /// Sets whether or not an event's thread id is displayed.
```

Comment:
How about
```suggestion
    /// Sets whether or not the [thread ID] of the current thread is displayed
    /// when formatting events 
    ///
    /// [thread ID]: https://doc.rust-lang.org/stable/std/thread/struct.ThreadId.html
```

Reply:
Also, it might be nice to show an example of what the output from this setting might look like...not a blocker though.